hedera-node/hapi-utils/src/test/java/com/hedera/node/app/hapi/utils/ethereum/EthTxDataTest.java

```diff
@@ -52,6 +54,9 @@ class EthTxDataTest {
     static final String EIP155_DEMO =
             "f86c098504a817c800825208943535353535353535353535353535353535353535880de0b6b3a76400008025a028ef61340bd939bc2195fe537567866003e1a15d3c71ff63e1590620aa636276a067cbe9d8997f761aecb703304b3800ccf555c9f3dc64214b297fb1966a3b6d83";
 
+    static final String EIP_2930_ADDRESS = "2c5d9eaed6ccb0705676b1ffc705507a1e0ea394";
+    static final String EIP_2930_PUBKEY = "023db57466d8612dae504043f8df82311b7681e1b93e0d55d893d8025ea94c0595";
```

Comment:
We shouldn't be introducing new public addresses for test data, tracking the keys is hard.

Use the 0xa94f key, it's [privkey](https://github.com/ethereum/tests/blob/06e276776bc87817c38f6efb492bf6f4527fa904/src/Templates/TestEachOpcode/testOpcodesGen.js#L1037) is widely know.